servers/zms/src/main/java/com/yahoo/athenz/zms/ZMSImpl.java

```diff
@@ -2607,6 +2607,53 @@ public Template getTemplate(ResourceContext ctx, String templateName) {
         return template;
     }
 
+    @Override
+    public DomainTemplateDetailsList getDomainTemplateDetailsList(ResourceContext ctx, String domainName) {
+        DomainTemplateDetailsList domainTemplateDetailsList = null;
```

Comment:
we should move this down before line 2630 to be consistent with all other methods in the class.

Reply:
Make sense Done.